data-prepper-main/build.gradle

```diff
@@ -52,3 +53,32 @@ artifacts {
     allDependencyJars(jar)
 }
 
+
+jib {
```

Comment:
It would be nice to use jib only from the :release:docker project. Is that possible?